src/logging.c

```diff
@@ -1151,6 +1153,12 @@ rcutils_ret_t rcutils_logging_format_message(
 # define COLOR_GREEN 2
 # define COLOR_YELLOW 6
 # define IS_STREAM_A_TTY(stream) (_isatty(_fileno(stream)) != 0)
+#elif defined(__ZEPHYR__)
+# define COLOR_NORMAL 0
+# define COLOR_RED 0
+# define COLOR_GREEN 0
+# define COLOR_YELLOW 0
```

Comment:
Are those colors required?

Reply:
I'll look again, but I thought that the code required them to exist (even on a platform like zephyr, where they wouldn't be meaningful).

Reply:
If you don't define those color constants, the build fails. Now it's possible that you could go further and wrap all of the references to the constants in "#if !defined(__ZEPHYR__)" too, but that seems like the more complicated option considering that just defining them to be 0 works. 

It's your codebase, though, so I'll handle it however you wish